plugin/init.lua

```diff
@@ -369,30 +391,102 @@ function pub.fuzzy_load(window, pane, callback, opts)
 		end
 	end
 
-	local function insert_choices(type, fmt)
-		for _, file in ipairs(wezterm.glob("*", pub.save_state_dir .. "/" .. type)) do
-			local label
+	local function file_datetime(folder, file)
+		-- Month names
+		local months = {
+			"Jan",
+			"Feb",
+			"Mar",
+			"Apr",
+			"May",
+			"Jun",
+			"Jul",
+			"Aug",
+			"Sep",
+			"Oct",
+			"Nov",
+			"Dec",
+		}
+		local result = ""
+		if is_windows then
+			local handle = io.popen("wmic datafile where name='" .. folder .. separator .. file("' get LastModified"))
+			if handle then
+				local stdout = handle:read("*a")
+				handle:close()
+
+				-- Convert the date format into "dd-mmm-yyyy HH:MM"
+
```

Comment:
For your consideration: I prefer to show date either as yyyy-mm-dd or as dd-mmm-yyyy to prevent day/month confusion as to the locale, these always work (obviously) while dd-mm is always prone to confusion between the US and Europe. But this is your project, I'll proceed as you prefer

Reply:
I am EU based, so I prefer the `dd-mm-yyyy HH:MM` myself as a default. 
If this becomes an issue later on, then i think it would be easy to add an user specified option to change the date formatting. But I think it is fine to leave it as is.

The suggested change only removes the 3'rd `m` in the month of the format

Reply:
Also changing the comment won't change the code which is hard coded to provide this format. I'm no lua expert, you might have a better way to do that, e.g. we could put that in the options and let the user choose its date format and if lua has a date/time formatting function we can rely on we can use it. Wdyt?

Reply:
I am not aware that you can get the date format from lua itself, so the best option is to let the user choose the date format, and provide the default as `dd-mm-yyyy HH:MM`

Reply:
Ok, I'll try to figure out a better way to work on formatting dates and giving the user reasonable options.